src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -137,16 +138,15 @@ private static void usage() {
   public static void main(final String[] args) {
     ErrorConsole.getConsole();
 
-    try (PerfTimer timer = Perf.startTimer("Show main window")) {
-      // do after we handle command line args
-      checkForMemoryXMX();
-      SwingUtilities.invokeLater(() -> setupLookAndFeel());
-      showMainFrame();
-    }
-    (new Thread(() -> setupLogging())).start();
-
+    PerfTimer timer = PerfTimer.startTimer("Show main window");
+    // do after we handle command line args
+    checkForMemoryXMX();
+    SwingUtilities.invokeLater(() -> setupLookAndFeel());
+    showMainFrame();
+    new Thread(() -> setupLogging()).start();
+    timer.stop();
```

Comment:
As mentioned above, let's remove the timer, this way you don't have to worry about the try/finally idiom.